int64 track_id

BoundingBox2D[] face_roi
BoundingBox2D[] face_region
sensor_msgs/Image[] face_image
